src/backend/marsha/settings.py

```diff
@@ -262,6 +262,18 @@ class Base(Configuration):
     NB_DAYS_BEFORE_DELETING_LIVE_RECORDINGS = values.Value(14)
     NB_DAYS_KEEPING_LIVE_IDLE = values.Value(7)
 
+    # XMPP Settings
+    LIVE_CHAT_ENABLED = values.BooleanValue(False)
+    XMPP_BOSH_URL = values.Value(None)
+    XMPP_CONFERENCE_URL = values.Value(None)
+    XMPP_PRIVATE_SERVER_JID = values.Value(None)
+    XMPP_PRIVATE_SERVER_PORT = values.Value(5222)
+    XMPP_PRIVATE_SERVER_PASSWORD = values.Value(None)
+    XMPP_JWT_SHARED_SECRET = values.Value(None)
+    XMPP_JWT_ISSUER = values.Value("marsha")
+    XMPP_JWT_AUDIENCE = values.Value("marsha")
+    XMPP_JID = values.Value(None)
```

Comment:
I will add then in the documentation.

> What is a JID and the expected format for such a value ?

A JID is a Jabber id. It's a unique Identifier for a resource in an XMPP network: https://xmpp.org/extensions/xep-0029.html

> Should I set the value of the private virtualhost in XMPP_PRIVATE_SERVER_JID ?

It's the admin JID, the variable name is maybe not accurate here.

> What is the format of a conference URL ?

It should be a host, you think I should use a [URLValue](What is the format of a conference URL ?) ?

Reply:
Thanks! After reading your explanations, may I suggest the following: 

```suggestion
    # XMPP Settings
    LIVE_CHAT_ENABLED = values.BooleanValue(False)
    XMPP_BOSH_URL = values.Value(None)
    XMPP_CONFERENCE_DOMAIN = values.Value(None)
    XMPP_PRIVATE_ADMIN_JID = values.Value(None)
    XMPP_PRIVATE_SERVER_PORT = values.Value(5222)
    XMPP_PRIVATE_SERVER_PASSWORD = values.Value(None)
    XMPP_JWT_SHARED_SECRET = values.Value(None)
    XMPP_JWT_ISSUER = values.Value("marsha")
    XMPP_JWT_AUDIENCE = values.Value("marsha")
    XMPP_DOMAIN = values.Value(None)
```
A few comments on this:

- `XMPP_JID`: Based on the values that you are using in your tests, I found this name ambiguous. To use the vocabulary used in [XEP-0029](https://xmpp.org/extensions/xep-0029.html#sect-idm44926487457152), it seems that you expect a domain identifier instead of a JID. This settings is used for 2 purpose : 
    - To set the subject field of the JWT token: in this case, if I understand the [documentation](https://github.com/jitsi/lib-jitsi-meet/blob/master/doc/tokens.md#token-structure) correctly, a domain is expected.
    - To initialize the converse.js client: in this case, since it's an anonymous login, a domain name is also expected according to the [documentation](https://m.conversejs.org/docs/html/configuration.html#anonymous)
   
- `XMPP_CONFERENCE_URL`: it also seems to be a domain identifier, according to the XMPP vocabulary and [prosody documentation on components](https://prosody.im/doc/components#:~:text=Components%20are%20extra%20services%20on,or%20gateways%20to%20other%20protocols.).